commands/daemon.go

```diff
@@ -153,12 +153,12 @@ func getRepo(req *cmds.Request) (repo.Repo, error) {
 // A message sent to or closure of the `terminate` channel signals the server to stop.
 func RunAPIAndWait(ctx context.Context, nd *node.Node, config *config.APIConfig, ready chan interface{}, terminate chan os.Signal) error {
 	servenv := &Env{
-		blockMiningAPI: nd.BlockMiningAPI,
+		blockMiningAPI: nd.BlockMining.BlockMiningAPI,
 		ctx:            ctx,
 		inspectorAPI:   NewInspectorAPI(nd.Repo),
 		porcelainAPI:   nd.PorcelainAPI,
-		retrievalAPI:   nd.RetrievalAPI,
-		storageAPI:     nd.StorageAPI,
+		retrievalAPI:   nd.RetrievalProtocol.RetrievalAPI,
+		storageAPI:     nd.StorageProtocol.StorageAPI,
```

Comment:
These names stutter now. It might not immediately make sense, but we should move toward the exported references being like `nd.StorageProtocol.API`.

Reply:
Yes, agreed. I'll rename things on a separate PR.